packages/filters/src/Utils.ts

```diff
@@ -74,3 +74,43 @@ export function getLabelForBooleanFilter(filterType: TypeValue): string {
       throw new Error(`Unrecognized boolean filter type ${filterType}`);
   }
 }
+
+export function getSymbolForTextFilter(filterType: TypeValue): string {
```

Comment:
You should be able to just use `TableUtils.getFilterOperatorString` instead of defining a new function here. May need to add a couple cases (e.g. `eqIgnoreCase`).

Reply:
According to the `TableUtils` tests, any case not currently in the switch statement is supposed to throw. Luckily, for the purposes of the linker, I don't think we need to add any new cases except for `startsWith` and `endsWith` but that can just be handled via conditionals inside `Linker.tsx`.